draft-ietf-quic-transport.md

```diff
@@ -1469,9 +1469,17 @@ of three ways:
    should send a Public Reset packet in return.  (TODO: articulate rules around
    when a public reset should be sent.)
 
-TODO: Connections that are terminated are added to a TIME_WAIT list at the
-server, so as to absorb any straggler packets in the network.  Discuss TIME_WAIT
-list.
+After receiving either a CONNECTION_CLOSE frame or a Public Reset, an
+endpoint MUST NOT send additional packets on that connection. After
+sending either a CONNECTION_CLOSE frame or a Public Reset packet,
+implementations MUST NOT send any non-closing packets on that connection. If
+additional packets are received after this time, implementations
+SHOULD respond to them by sending either a CONNECTION_CLOSE frame or a
+Public Reset packet, either of which may just be a duplicate of a
```

Comment:
I agree with Ian that we don't need the endpoint to maintain complete state, so 2 and 3 make sense. I'd suggest that we add a suggestion about how long a peer resends CONNECTION_CLOSE after sending it once (basically how long state should be kept in option (2)), and I propose idle_timeout as a time period that makes sense.